src/WebUI/ClientApp/src/app/modules/member-management/member-list/member-list.component.html

```diff
@@ -30,15 +30,17 @@
 
       <ng-container matColumnDef="currentCareerLevel">
         <th mat-header-cell *matHeaderCellDef mat-sort-header>
-          Karriere Stufe
+          Karrierestufe
         </th>
         <td mat-cell *matCellDef="let element">
-          {{ element.currentCareerLevel }}
+          <p [title]="element.currentCareerLevel">
```

Comment:
```suggestion
          <span [title]="element.currentCareerLevel">
```
p puts the element up for some reason. It also needs some padding (you're the angular material expert, that should be possible right?)
![there isn't enough space](https://user-images.githubusercontent.com/66270203/159557267-91e47c33-75c5-45c6-ae00-b9218e80a4c7.png)